### PR TITLE
Readme improvements

### PR DESCRIPTION
**Pull request by sharonovd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,35 @@
 # Cartridge Command Line Interface
 
-## Managing instances
+## Installation
+
+Run
+```sh
+tarantoolctl rocks install cartridge-cli
```
Comment (rosik) [inline README.md]: `export PATH=$PWD/.rocks/bin/:$PATH`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,35 @@
 # Cartridge Command Line Interface
 
-## Managing instances
+## Installation
+
+Run
```
Comment (printercu) [inline README.md]: -

File: README.md [hunk 3]
```diff
@@ -1,6 +1,35 @@
...
+```sh
+tarantoolctl rocks install cartridge-cli
+```
+
+## Usage
+
+Run
```
Comment (printercu) [inline README.md]: \-

File: README.md [hunk 4]
```diff
@@ -1,6 +1,35 @@
...
+tarantoolctl rocks install cartridge-cli
+```
+
+## Usage
+
+Run 
+```sh
+cartridge --help
+```
+for more details
```
Comment (printercu) [inline README.md]: \-

File: README.md [hunk 5]
```diff
@@ -1,6 +1,35 @@
...
+
+### Applications lifecycle
+
+To create an application from template, say
```
Comment (printercu) [inline README.md]:
```suggestion
Create an application from template:
```

File: README.md [hunk 6]
```diff
@@ -1,6 +1,35 @@
...
+
+```sh
+cartridge create --name myapp
+```
+
+To pack an application, say
```
Comment (printercu) [inline README.md]:
```suggestion
Pack an application into distributable
```

**Pull request merged**